model/registry/file.yaml

```diff
@@ -36,3 +128,11 @@ groups:
         brief: >
           File size in bytes.
         stability: experimental
+      - id: file.target_path
```

Comment:
Since it's only applicable to symbolic links, having it explicit in the name would provide more clarity.

E.g. `file.symbolic_link.target` (`file.symbolic_link.target_path`) would be more descriptive.